src/main/java/org/jenkinsci/plugins/ewm/actions/ExwsAllocateActionWrapper.java

```diff
@@ -0,0 +1,51 @@
+package org.jenkinsci.plugins.ewm.actions;
+
+import hudson.model.Actionable;
+import hudson.model.Run;
+import jenkins.model.RunAction2;
+import org.jenkinsci.plugins.ewm.Messages;
+
+/**
+ * A {@link RunAction2} that acts as a wrapper to hold multiple {@link ExwsAllocateActionImpl}s.
+ *
+ * @author Alexandru Somai
+ */
+public class ExwsAllocateActionWrapper extends Actionable implements RunAction2 {
```

Comment:
If I use a `HashSet`, the set does not guarantee the order of the items.

Use case:  If the user calls `exwsAllocate` step multiple times, the set will contain multiple items. If I have a downstream job that uses this, it is stated that it should use the first registered `exwsAllocate` action. So, if a set is used, I don't have the guarantee which will be the first inserted item.
Therefore I would rather go for a `List`. Maybe a `LinkedList` implementation?